docs/development/release-checklist.md

```diff
@@ -0,0 +1,29 @@
+## Release Checklist
+
+- [ ] Update version number in package.json if not done after last release.
+- [ ] Tag release branch. Wait for the CI to build artifacts.
+- [ ] Sign windows installer.
+- [ ] Sign Linux installer.
+### Sign mac installer (As there's a issue with the zip produced by the build script, we need to manually build and zip, rename the file to replace space with dot etc )
+- [ ] Make sure the required env variables are set for the notorize, signing process.
+- [ ] git clean, reset to make sure a clean (CI equivalent) build.
+- [ ] Manually zip the installer.
+- [ ] Rename installer filename to replace space with dot.
+
+### Release Documentation
+- [ ] Release notes. Update on the Github draft Release page.
+- [ ] docs update (Help, Readme..)
+- [ ] Slack Announcements
+- [ ] Newsletter summary
+- [ ] Update metrics, roadmap on Confluence page
```

Comment:
These are internal things. Should that be in the public release material? I wouldn't put internal things in here.

Reply:
Removed marketing section.